frontend/src/components/ShoppingListDetail.tsx

```diff
@@ -18,6 +19,16 @@ export default function ShoppingListDetails() {
             });
     }, [id]);
 
+    const fetchShoppingList = () => {
```

Comment:
Braucht man die fetchShoppingList? Reicht der API Aufruf im useEffect nicht? 

Statt setItems und fetchShoppingList könnte man dann einfach: `setList({ ...list, item: updatedItems })`  in der handleUpdateDone aufrufen.  
Ebenfalls könnte dann der items useState entfernt werden.

Reply:
Lass uns hierzu nochmal sprechen am Montag